app/interfaces/order.ts

```diff
@@ -0,0 +1,249 @@
+declare namespace IOrder {
+  export interface Link {
+    rel: string;
+    uri: string;
+  }
+
+  export interface PreferredName {
+    uuid: string;
+    display: string;
+    links: Link[];
+  }
+
+  export interface PreferredAddress {
+    uuid: string;
+    display: string;
+    links: Link[];
+  }
+
+  export interface Attribute {
+    uuid: string;
+    display: string;
+    links: Link[];
+  }
+
+  export interface Link4 {
+    rel: string;
+    uri: string;
```

Comment:
```suggestion
  export interface Link4 {
    rel: string;
    uri: string;
```
Similar to interface already declared as `Link`